src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfigTest.java

```diff
@@ -129,6 +129,36 @@ public void testValidateTimePartitioningColumnWithNullAndDate() throws
     Assert.assertEquals(0, collector.getValidationFailures().size());
   }
 
+  @Test
+  public void testValidateTimePartitioningColumnWithMonthAndDateTime() throws
+    InvocationTargetException, IllegalAccessException {
+
+    String columnName = "partitionFrom";
+    Schema schema = Schema.of(Schema.LogicalType.DATETIME);
+
+    Schema fieldSchema = schema.isNullable() ? schema.getNonNullable() : schema;
+    TimePartitioning.Type timePartitioningType = TimePartitioning.Type.MONTH;
+
+    validateTimePartitioningColumnMethod.invoke(config, columnName, collector, fieldSchema, timePartitioningType);
+    // No error as null time timePartitioningType will default to DAY
```

Comment:
This comment does not sound right, we did pass the `timePartioningType` while calling the method, why will it be null in this case?

Reply:
It did make sense [here](https://github.com/data-integrations/google-cloud/blob/f1578981abad2b4220e5e55fb421a6efda5c10c9/src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfigTest.java#L128) because `timePartitioningType` was passed as `null` there.

Reply:
Yup it's a simple copy pasting error, will fix this!

Reply:
Removed, this comment !